tests/bugfixes/redmine/test_issue_1043.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+import string
+
+import system_tests
+
+
+class FailureOnCifsShares(metaclass=system_tests.CaseMeta):
```

Comment:
It is passing now, I made a mistake when configuring it. But you are right that we are actually not testing it on the CI in a meaningful way. Nevertheless, I'd keep it as a regression test, in case someone runs it on a CIFS share.